packages/core/src/execution-engine/__tests__/workflow-execute.test.ts

```diff
@@ -555,11 +555,17 @@ describe('WorkflowExecute', () => {
 			);
 
 			// ASSERT
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+			const runNodeFilter: string[] = (workflowExecute as any).runExecutionData.startData
```

Comment:
Not sure what's best practice, but I typically prefer `ts-expect-error` to access private fields rather than introducing `any`.